### DISCUSSION TITLE
Unable to access mitm.it

### DISCUSSION THREAD

**Discussion by axxturelmeatrider** [mitmproxy/mitmproxy#7091] [Q&A]
To run a program I have it requires the certificate from mitm.it and the proxy settings I have enabled won't let me access the site. I have attached the settings I have down below and yes I am running mitmproxy through command prompt but I haven't done anything with that and idk if I have to to get mitm.it to work. Please lmk if I am doing anything wrong, I'm a complete noob so please, only answers or helpful inquiries.
![image](https://github.com/user-attachments/assets/7f238345-55b5-45d7-ba58-219ef786958e)

**Comment by trackme518** [comment]
Hi, in my case what helped - type specifically http://mitm.it - note that it is not HTTPS
